.pipelines/verifyBuildRequest.ps1

```diff
@@ -0,0 +1,13 @@
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$True,Position=1)]
+    [string]$commit
+)
+
+$gitHubCommit = Invoke-RestMethod -Method Get "https://api.github.com/repos/microsoft/PowerToys/commits/$commit"

```

Comment:
Do we need to make changes here too?

Reply:
@jaimecbernardo - Thanks. I nuked this since it's unneeded. I went the route of running the Pull checks with an inline script that doesn't require doing a checkout of the repo if not needed. That was leftover, so I removed it in a commit I just added.